src/lib/container/list/list.stories.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+	import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+	import type { ListOptions, ListTransform } from '$lib/container/list/list';
+	import List from './list.svelte';
+	import type { MockListElement } from './list.mocks';
+	import { mockListElements } from './list.mocks';
+	import { ListStyle } from '$lib/theme/theme';
+
+	type ResultElement = MockListElement & { output: string };
+	const customTransform: ListTransform<ResultElement, MockListElement> = (
```

Comment:
move to [src/lib/container/list/list.mocks.ts](https://github.com/nahuelio/nahuelio.github.io/pull/1/files#diff-6ca902a717a4526a9e37e9ebab30dd69aa1911be4e225262ba2a131cde734c0b)